Bottom: ecea98adba28a4fd3e38edc4505c5c8f5d4a7d46
Top:    9a5e9b6a56f09680f8110c4fe8aa17cdcefc7cca
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-02 00:39:58 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
